en/advanced_config/tuning_the_ecl_ekf.md

```diff
@@ -273,6 +279,19 @@ Range finder distance to ground is used by a single state filter to estimate the
 
 If operating over a flat surface that can be used as a zero height datum, the range finder data can also be used directly by the EKF to estimate height by setting the [EKF2_HGT_MODE](../advanced_config/parameter_reference.md#EKF2_HGT_MODE) parameter to 2.
 
+#### Range Finder Blockage Detection
+
+When not used as the primary height source, blockage is detected using a kinematic consistency check between the vertical velocity estimate and the numerical derivative of the range finder data.
```

Comment:
@bresch What actually is "blockage"? The word implies that the rangefinder is used to determine that the vehicle can't descend at a desired rate for some reason. But then we're hovering, so not clear to me what this is doing.